### ISSUE TITLE
[Bug] Listing buttons fail a11y testing

### ISSUE THREAD

**Issue by pfeeny5** [CityOfDetroit/bloom#340]
[REMEMBER: a bug should be understandable years later by someone who will never talk to the reporter or assignee.]

STEPS TO REPRODUCE:
1. Run a Lighthouse accessibility report on the Listings List page
2. Expand "Background and foreground colors do not have a sufficient contrast ratio." issue for offending elements

EXPECTED RESULTS:
All elements are expected to pass

OBSERVED RESULTS:
"Filter Listings" button, "See Details" button, and "Application Due" banner are recommended to improve contrast

ADDITIONAL INFORMATION:

**Comment by avaleske** [comment]
This might not be a problem with future color scheme changes. Unassigning myself for now.

**Comment by jnowell** [comment]
Even with new redesign, this still fails testing for the 'Filter' button and the 'See Details' button.

**Comment by jnowell** [comment]
The 'Next' and 'Previous' buttons will also need to be updated.
